### PR TITLE
Improvement/adjust the add post section

### PR DESCRIPTION
**Pull request by celinelemenn**

### CODE REVIEW

File: docs/Gemfile [hunk 1]
```diff
@@ -7,3 +7,4 @@ group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.6"
   gem 'jekyll-include-cache'
 end
+gem 'faraday', '0.17.3'
```
Comment (lvonk) [inline docs/Gemfile]: Missing newline

File: docs/docs/finishing-the-web-application.md [hunk 2]
```diff
@@ -423,6 +443,32 @@ The complete `app/views/authors/show.erb`
...
+
+```ruby
+  class AuthorRecord < Sequent::ApplicationRecord
+    has_many :post_records, foreign_key: 'author_aggregate_id', primary_key: 'aggregate_id'
+  end
+```
+
+We need to add this new foreign_key as a new column in the post table. We need to update `db/tables/post_records.sql`
+                                                                                                                                                
+```ruby
```
Comment (lvonk) [inline docs/docs/finishing-the-web-application.md]: This is `sql`

File: docs/Gemfile (hunk 1 above)
Comment (celinelemenn) [reply docs/Gemfile]: added a new line before

File: docs/docs/finishing-the-web-application.md (hunk 2 above)
Comment (celinelemenn) [reply docs/docs/finishing-the-web-application.md]: changed for sql

**Pull request merged**